completion_aggregator/api/v0/views.py

```diff
@@ -0,0 +1,353 @@
+"""
+API views to read completion information.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from rest_framework.exceptions import NotAuthenticated, NotFound, ParseError, PermissionDenied
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework import status
+
+from django.contrib.auth import get_user_model
+
+from courseware.courses import get_course
+from opaque_keys.edx.keys import CourseKey, UsageKey
+from openedx.core.lib.api import authentication, paginators
```

Comment:
Any code from edx platform (courseware and openedx fall into that category) will need to be shimmed in the compat modules, as described in my other comment.